packages/extension/src/shared/transactions.ts

```diff
@@ -5,17 +5,19 @@ import { ReviewTransactionResult } from "./actionQueue/types"
 import { WalletAccount } from "./wallet.model"
 import { AlephiumExplorerTransaction } from "./explorer/type"
 
-export type Status = 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_CHAIN' | 'REJECTED';
+export type Status = 'NOT_RECEIVED' | 'RECEIVED' | 'PENDING' | 'ACCEPTED_ON_MEMPOOL' | 'ACCEPTED_ON_L2' | 'ACCEPTED_ON_CHAIN' | 'REJECTED';
```

Comment:
Do we need to remove the `ACCEPTED_ON_L2`?

Reply:
Yes, but it will break a lot of other code. So we keep them for now.